android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageViewModel.kt

```diff
@@ -105,19 +109,28 @@ class CreateListingPageViewModel @Inject constructor(
 						}
 					}.awaitAll()
 
+					val location = locationService.getLocation()
 					listingsApi.listingsCreate(
 						CreateListingRequest(
 							addressLine = it.address.addressLine,
 							addressCity = it.address.addressCity,
 							addressPostalcode = it.address.addressPostalCode,
 							addressCountry = it.address.addressCountry,
 							price = it.price,
-							rooms = it.numBedrooms,
 							leaseStart = it.startDate,
 							leaseEnd = it.endDate,
 							description = it.description,
 							residenceType = ResidenceType.house,
 							imgIds = imgIds,
+							latitude = location!!.latitude.toFloat(),
```

Comment:
Can we maybe not take down the entire app if there is no location?

Reply:
Moving to separate PR.